### DISCUSSION TITLE
Lingui Extract <Files> seems to ignore whatever Path I provide

### DISCUSSION THREAD

**Discussion by krystiankolup** [lingui/js-lingui#2023] [Q&A]
Hi I'm having issues with the lingui extract command. Im running lingui extract src/Scenes/Unit/. It just seems to still follow the path specified in my lingui Config rather than the one Ive provided. Any tips ?

**Comment by timofei-iatsenko** [comment]
AFAIK it should filter whatever would be matched from lingui config. It means if you pass a completely different path, it will match zero results.

**Reply by krystiankolup** [reply]
is there no way of specifying a specific file I have changed and get the extractor to only extract translations from that one file without doing the matching from the lingui config? As running that takes a really long time on my system whereas If i specify the file it would be instant

**Reply by timofei-iatsenko** [reply]
It should filter whatever you specified in the lingui.config. For example: 

If in lingui.config you have "./moduleA/**" and in path you have "./moduleB" this will not work, and will return an empty result. 

But  "./moduleA/my-component.jsx" should work. And extract from only one file.
